pkg/channel/event_dispatcher.go

```diff
@@ -80,16 +78,16 @@ type EventDispatcher struct {
 
 // NewEventDispatcher creates a new event dispatcher that can dispatch
 // events to HTTP destinations.
-func NewEventDispatcher(logger *zap.Logger) *EventDispatcher {
+func NewEventDispatcher(logger *zap.Logger, config EventDispatcherConfig) *EventDispatcher {
```

Comment:
The Kafka dispatcher is using this function. Maybe keep the original function signature calling the new function with the default values, and open an issue against eventing-contrib.